src/typ/prelude.rs

```diff
@@ -1,7 +1,6 @@
 use super::{Environment, Type, TypeConstructor, ValueConstructorVariant};
 use crate::error::GleamExpect;
 use std::sync::Arc;
-
```

Comment:
Could you revert the changes in this file please? Seems all the blank lines have been removed.

Reply:
I just ran `cargo fmt`... Is that not a part of the CI check?

Reply:
You could add a rustfmt.toml if you have specific preferences about the way it's formatted.

Reply:
It is part of the CI check and by default it will keep these lines. The blank lines have been kept in the other files so I'm not sure this is the result of cargo fmt?